iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/import_/Importer.java

```diff
@@ -55,14 +56,26 @@
      */
     boolean exists(ReadOnlyDataSource dataSource);
 
+
+    /**
+     * @deprecated Use {@link Importer##importData(ReadOnlyDataSource, NetworkFactory, Properties)} instead.
```

Comment:
Is the javadoc référence ok? (two dashes)

Reply:
Right, link was broken. Fixed.